raft-tech/TANF-app
# Issue 3033: Updating range for FQ validator

## Summary of Changes

Pull request closes #3033 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ or [deployed env](https://tdp-frontend-raft.app.cloud.gov) and sign in.
1. submit TANF/Tribal/SSP correct section 3 and/or  4 files for FY21Q1 and confirm that no FY-related errors are present in error report. 
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.

_test files for FY21Q1_
[2021.Q1.Tribal Aggregate Data.txt](https://github.com/user-attachments/files/16473061/2021.Q1.Tribal.Aggregate.Data.txt)
[2021.Q1.Aggregate Data.txt](https://github.com/user-attachments/files/16473095/2021.Q1.Aggregate.Data.txt)
[2021.Q1.SSP Aggregate Data.txt](https://github.com/user-attachments/files/16473140/2021.Q1.SSP.Aggregate.Data.txt)

> *Demo GIF(s) and screenshots for testing procedure*

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] `Year 2020 must be larger than 2020.` is not raised for Tribal and TANF T6 records where CALENDAR_QUARTER == 20204. 
- [x] `Year 2020 must be larger than 2020.` is not raised for Tribal and TANF T7 records where CALENDAR_QUARTER == 20204. 
- [x] `Year 2020 must be larger than 2020.` is not raised for SSP M6 records where CALENDAR_QUARTER == 20204. 
- [x] `Year 2020 must be larger than 2020.` is not raised for SSP M7 records where CALENDAR_QUARTER == 20204.  

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
